jaxlib/mosaic/gpu/custom_call.cc

```diff
@@ -382,19 +485,21 @@ void DumpCompilationOutput(mlir::ModuleOp module, const std::string& sm,
       ptxas_args.push_back("-v");
     }
     ptxas_args.push_back(nullptr);
-    if (auto status = RunCUDATool("ptxas", ptxas_args); !status.ok()) {
-      std::cerr << "ptxas invocation failed: " << status.message() << std::endl;
+    if (auto result = RunCUDATool("ptxas", ptxas_args); !result.ok()) {
+      std::cerr << "ptxas invocation failed: " << result.status() << std::endl;
       continue;
     }
     if (!dump_sass) { continue; }  // We're done.
     // Call nvdisasm to pretty-print SASS.
-    if (auto status = RunCUDATool(
-            "nvdisasm", {"nvdisasm", "-ndf", "-c", elf_path.c_str(), nullptr});
-        !status.ok()) {
-      std::cerr << "nvdisasm invocation failed: " << status.message()
+    auto result = RunCUDATool(
+        "nvdisasm", {"nvdisasm", "-ndf", "-c", elf_path.c_str(), nullptr});
+    if (!result.ok()) {
+      std::cerr << "nvdisasm invocation failed: " << result.status()
                 << std::endl;
       continue;
     }
+    // Dump SASS.
+    std::cout << *result << std::endl;
```

Comment:
I think you missed an explicit print for ptxas output above (if dump_ptxas is on)